### ISSUE TITLE
Events are not tacked after update to v3.6.8

### ISSUE THREAD

**Issue by mihai8804858** [segmentio/analytics-ios#733]
Updating Segment to `v3.6.8` led to all events not being tracked in Segment Debugger.
After some debugging we found that payload structure has changes which can be the reason Segment is not able to parse the events.

`v3.6.7`:
```
{
    "batch": [{
        "messageId": ...,
        "anonymousId": ...,
        "userId": ...,
        "context": {... },
        "properties": {...},
        "type": ...,
        "timestamp": ...,
        "name": ...,
        "integrations": {}
    }, {
        "messageId": ...,
        "anonymousId": ...,
        "userId": ...,
        "event": ...,
        "context": {...},
        "properties": {...},
        "type": ...,
        "timestamp": ...,
        "integrations": {...}
    }],
    "sentAt": ...
}
```

`v3.6.8`
```
[{
        "messageId": ...,
        "anonymousId": ...,
        "userId": ...,
        "context": {...},
        "properties": {... },
        "type": ...,
        "timestamp": ...,
        "name": ...,
        "integrations": {...}
    }, {
        "messageId": ...,
        "anonymousId": ...,
        "userId": ...,
        "event": ...,
        "context": {... },
        "properties": {...},
        "type": ...,
        "timestamp": ...,
        "integrations": {...}
    },
    "sentAt": ...
]
```

**Comment by f2prateek** [comment]
Let me check, and get back to you.

**Comment by f2prateek** [comment]
hey @mihai8804858, we just fixed the bug in #736 . The fix is being released in 3.6.9 now.

Thanks for reporting the issue and sorry about this bug - we're taking this seriously and will be making significant improvements to our testing suite to avoid these kinds of issues again in the future.

**Comment by mihai8804858** [comment]
Thank you @f2prateek
